### PR TITLE
Feat/sig-6

### PR DESCRIPTION
**Pull request by kavyamehta**
Finished creating frontend settings page. 

ACCEPTANCE CRITERIA:Upon clicking the settings icon on the main page, user should be directed to a separate page called Settings. This page should show the user the different controls such as the light/dark mode, size of text, and theme colour.

### CODE REVIEW

**Review by Blizzard416 — APPROVED** [review]
All good :)

File: frontend/lib/main.dart [hunk 1]
```diff
@@ -1,123 +1,17 @@
...
-  800:Color.fromRGBO(17,138,126, .9),
-  900:Color.fromRGBO(17,138,126, 1),
-};
-
 class MyApp extends StatelessWidget {
   const MyApp({Key? key}) : super(key: key);
 
-  // This widget is the root of your application.
-  @override
-  Widget build(BuildContext context) {
```
Comment (milindvishnoi) [inline frontend/lib/main.dart:27]: Removing this part removes the light/dark mode stuff. Please fix this!

File: frontend/lib/main.dart [hunk 2]
```diff
@@ -1,123 +1,17 @@
 import 'package:flutter/material.dart';
+import 'package:frontend/settings.dart';
 
 void main() {
   runApp(const MyApp());
 }
 
-// Custom Green for app
```
Comment (milindvishnoi) [inline frontend/lib/main.dart:8]: This is the custom green of our app.

File: frontend/lib/settings.dart [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+                Icons.logout,
+                color: Colors.teal,
+                size: 24.0,
+              ),
+              SizedBox(width: 5,),
+              Align(
+                alignment: Alignment.centerLeft,
+                child: Text(
+                  "Log out",
+                  style: TextStyle(fontWeight: FontWeight.w900, color: Colors.teal),
```
Comment (milindvishnoi) [inline frontend/lib/settings.dart]: Please use the custom color, mentioned above.

**Review by milindvishnoi — CHANGES_REQUESTED** [review]
The settings page looks greatt!

**Comment by milindvishnoi** [comment]
Everything seems to be working fine. Great job!

**Pull request merged**